typedb/stream/bidirectional_stream.py

```diff
@@ -100,6 +100,9 @@ def _collect(self, response: Union[transaction_proto.Transaction.Res, transactio
         else:
             raise TypeDBClientException.of(UNKNOWN_REQUEST_ID, request_id)
 
+    def drain_errors(self) -> [RpcError]:
```

Comment:
Is this a valid type hint? I've always used `List[RpcError]`

Reply:
HMMM i dunno, i should conform to our other styles